pex/pip/tool.py

```diff
@@ -631,63 +630,22 @@ def spawn_install_wheel(
                     "platform.".format(wheel, interpreter)
                 )
 
-        install_cmd = [
-            "install",
-            "--no-deps",
-            "--no-index",
-            "--only-binary",
-            ":all:",
-            # In `--prefix` scheme, Pip warns about installed scripts not being on $PATH. We fix
-            # this when a PEX is turned into a venv.
-            "--no-warn-script-location",
-            # In `--prefix` scheme, Pip normally refuses to install a dependency already in the
-            # `sys.path` of Pip itself since the requirement is already satisfied. Since `pip`,
-            # `setuptools` and `wheel` are always in that `sys.path` (Our `pip.pex` venv PEX), we
-            # force installation so that PEXes with dependencies on those projects get them properly
-            # installed instead of skipped.
-            "--force-reinstall",
-            "--ignore-installed",
-            # We're potentially installing a wheel for a foreign platform. This is just an
-            # unpacking operation though; so we don't actually need to perform it with a target
-            # platform compatible interpreter (except for scripts - which we deal with in fixup
-            # install below).
-            "--ignore-requires-python",
-            "--prefix",
-            install_dir,
-        ]
+        return spawn_python_job(
```

Comment:
Eventually I'll remove wheel installation completely out of the Pip class here since the code no longer relies on Pip at all. This is just the refactor step to set that up where I didn't want to introduce too much noise.